dialogue-clients/src/main/java/com/palantir/dialogue/clients/ReloadingClientFactory.java

```diff
@@ -382,105 +456,106 @@ private ClientConfiguration hydrate(ClientConfiguration configuration) {
         return configuration;
     }
 
-    // TODO(ckozak): Each of the 'map(' blocks should be extracted to another method. I'm not doing that
-    // yet be cause it will make the diff much harder to review.
-    @SuppressWarnings("CyclomaticComplexity")
+    private Refreshable<InternalDialogueChannelConfiguration> getServiceRefreshable(String serviceName) {
```

Comment:
minor nit: i'm not wild about this method name, perhaps something like `configurationForService` as this returns a configuration that can be used to create a channel for this service? not super picky though

Reply:
👍 One of the two hardest problems in CS